bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java

```diff
@@ -173,13 +173,23 @@
             .withDescription("Controls the color of the light").withCategory("ColorLight").build();
 
     /**
-     * Color-temperature: default system wide {@link ChannelType} which allows changing the color temperature
+     * Color-temperature: default system wide {@link ChannelType} which allows changing the color temperature in percent
      */
     public static final ChannelType SYSTEM_COLOR_TEMPERATURE = ChannelTypeBuilder
             .state(new ChannelTypeUID(BINDING_ID, "color-temperature"), "Color Temperature", "Dimmer")
-            .withDescription("Controls the color temperature of the light").withCategory("ColorLight")
-            .withStateDescriptionFragment(StateDescriptionFragmentBuilder.create().withMinimum(BigDecimal.ZERO)
-                    .withMaximum(new BigDecimal(100)).withPattern("%d").build())
+            .withDescription("Controls the color temperature of the light from 0 (cold) to 100 (warm)")
+            .withCategory("ColorLight").withStateDescriptionFragment(StateDescriptionFragmentBuilder.create()
+                    .withMinimum(BigDecimal.ZERO).withMaximum(new BigDecimal(100)).withPattern("%d").build())
+            .build();
+
+    /**
+     * Color-temperature: default system wide {@link ChannelType} which allows changing the color temperature in Kelvin
+     */
+    public static final ChannelType SYSTEM_COLOR_TEMPERATURE_ABS = ChannelTypeBuilder
+            .state(new ChannelTypeUID(BINDING_ID, "color-temperature-abs"), "Color Temperature", "Number")
```

Comment:
I‘m unsure if we should support UoM here. In general it is a quantity with a unit. OTOH I have never seen someone using another unit than K.

Reply:
Nobody is sure here, my tendency goes to what @cweitkamp did here, see https://github.com/openhab/openhab-addons/pull/9939#discussion_r563350083.

Reply:
I was thinking the same, that’s why it is implemented in deconz that way. The only other unit I know is mired (which is used in Zigbee internally), but I have never seen it user-facing. So a plain `Number` is fine with me, too.